editor/pot_generator.cpp

```diff
@@ -182,12 +183,12 @@ void POTGenerator::_write_msgid(Ref<FileAccess> r_file, const String &p_id, bool
 		r_file->store_line("\"\"");
 	}
 
+	// Escaping `'` is unnecessary, and is incompatible with translation tools like Weblate.
```

Comment:
Shouldn't this be line below, for better context?

Reply:
It's explaining the reason for the `replace()` function in both blocks below.

Reply:
It does, but currently it may look like it applies to the `for` loop, not the line in loop.

Reply:
So you think it would be better to just have the comment in one of the blocks?

Reply:
IMO yes.

Reply:
Done.